### ISSUE TITLE
How did this work out?

### ISSUE THREAD

**Issue by robwolff3** [cpk123/tank-level#1]
Hey @cpk123 ,

I found your post over on the raspberrypi.com forums with similar micropython code. I'm tying to use your code to read three seelevel tank sensors that came with my RV. Did you ever deploy this in your RV and have you made any changes to the code since?

Thank you for your efforts on this. I'm excited to get it running in mine.
Rob

**Comment by cpk123** [comment]
No, for various reasons I did not actually use this in my Revel. It did work on the bench, though. But, if I were going to deploy, I would recode in C so that timing would not be an issue. I would not claim that the micropython is rock solid, since there is no real field experience or burn-in for this code.
    On Wednesday, May 11, 2022, 05:31:36 AM PDT, Rob Wolff ***@***.***> wrote:  
 
 


Hey @cpk123 ,

I found your post over on the raspberrypi.com forums with similar micropython code. I'm tying to use your code to read three seelevel tank sensors that came with my RV. Did you ever deploy this in your RV and have you made any changes to the code since?

Thank you for your efforts on this. I'm excited to get it running in mine.
Rob
